helm/cloud-provider-cloud-director-app/charts/cloud-provider-for-cloud-director/values.yaml

```diff
@@ -0,0 +1,26 @@
+configmap:

```

Comment:
more meaningful label here would be nice

Reply:
changed to cpiConfigmap